Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.9.6-slim-buster as parent
+
+RUN apt-get update && apt-get install -y curl && \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python && \
+    echo 'export PATH="$HOME/.poetry/bin:$PATH"' >> $HOME/.profile && \
+    /bin/bash -c "source $HOME/.profile"
+
+ENV PATH="${HOME}/.poetry/bin:${PATH}"
```

Comment:
Minor, but I don't think this is working because [the $HOME variable isn't available here](https://github.com/moby/moby/issues/28971)  - instead we could specify this directly;
`ENV PATH="/root/.poetry/bin:${PATH}"`
With that command working, we could lose lines 5&6, as well as reducing our poetry commands from `$HOME/.poetry/bin/poetry ...` to just `poetry ...`

Reply:
Thanks, done